modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.js

```diff
@@ -151,7 +151,9 @@ const FrontendDataSet = ({
 			...initialActiveView,
 		};
 
-		const filters = initialFilters.map((filter) => {
+		const filters = [];
+
+		initialFilters?.forEach((filter) => {
```

Comment:
How come you switched from map to a forEach+push? (I typically find map easier to reason through than a forEach+push.

For checking null, you could do `initialFilters ? initialFilters.map(... : []`

Reply:
^ non blocker fyi

Reply:
Yeah, that looks better, I'll make the change. `forEach` somewhat implies splitting of the array, where here we modify and return the same number of items.